README.md

```diff
@@ -1,2 +1,8 @@
-# git_demo_2018
-Forking, Pushing and PRs
+# This is my new changes to the README
+
+- In this Demo I will
+- Fork a new repo
```

Comment:
Why so serious?

Reply:
Well its soon time to go home and im kinda hungry, but i will make the changes!